packages/web/app/src/components/ui/date-range-picker.tsx

```diff
@@ -273,13 +273,32 @@ export function DateRangePicker(props: DateRangePickerProps): JSX.Element {
   }, [quickRangeFilter, validUnits]);
 
   presets = [...presets, ...dynamicPresets].sort((a, b) => {
-    if (a.label < b.label) {
-      return -1;
+    const aWeight = a.label.includes('minutes')
```

Comment:
hmm, I guess you can move this logic of calculating weight into something else? 

Also, maybe the ideal way to understand the "weight" is to measure how many minutes you have between now and the result date?

For example:
"last 10 minutes" = 10 
"last 2 days" =  2880